ff/src/biginteger/arithmetic.rs

```diff
@@ -104,3 +104,62 @@ pub fn find_wnaf(num: &[u64]) -> Vec<i64> {
 
     res
 }
+
+pub fn find_relaxed_naf(num: &[u64]) -> Vec<i64> {
```

Comment:
Can the relaxed naf definition be commented here

Reply:
Yes, let me add some description to it.

Reply:
Added.